e2e/smoke_test.go

```diff
@@ -219,96 +227,116 @@ var _ = Describe("Smoke End To End Tests - against OpenShift Cluster with Istio
 			})
 
 			It("should watch for changes in ratings service in specified namespace and serve it", func() {
-				verifyThatResponseMatchesModifiedService(tmpDir, namespace)
+				EnsureAllPodsAreReady(namespace)
+				EnsureProdRouteIsReachable(namespace, ContainSubstring("ratings-v1"))
+
+				// given we have details code locally
+				CreateFile(tmpDir+"/ratings.rb", PublisherRuby)
+
+				ike := RunIke(tmpDir, "develop",
+					"--deployment", "ratings-v1",
+					"--port", "9080",
+					"--method", "inject-tcp",
+					"--watch",
+					"--run", "ruby ratings.rb 9080",
+					"--route", "header:x-test-suite=smoke",
+				)
+				EnsureAllPodsAreReady(namespace)
+				EnsureSessionRouteIsReachable(namespace, ContainSubstring("PublisherA"))
+
+				// then modify the service
+				modifiedDetails := strings.Replace(PublisherRuby, "PublisherA", "Publisher Ike", 1)
+				CreateFile(tmpDir+"/ratings.rb", modifiedDetails)
+
+				EnsureSessionRouteIsReachable(namespace, ContainSubstring("Publisher Ike"))
+
+				StopIke(ike)
```

Comment:
`Stop(ike)`?

Reply:
It sounds nicer, but then it doesn't match RunIke. Then again Stop is general...

Reply:
That's my point - it is general-purpose function. But also `StopIke(ike)` reads a bit weird. All in all it is just a minor thing, so we can leave it as is for now.

Reply:
You're right. Updated.